bin/concurrently.spec.ts

```diff
@@ -82,8 +80,8 @@ const run = (args: string, ctrlcWrapper?: boolean) => {
     });
 
     const exit = Rx.firstValueFrom(
-        Rx.fromEvent(child, 'exit').pipe(
-            map((exit: [number | null, NodeJS.Signals | null]) => {
+        Rx.fromEvent<[number | null, NodeJS.Signals | null]>(child, 'exit').pipe(
```

Comment:
Sorry, I realise this is more trouble for #343.
I think we could get away with a `resultSelector` arg that casts the value though

Reply:
No problem! We can address this in #343. We probably have to rely on `map` though as `resultSelector` is deprecated: https://rxjs.dev/deprecations/resultSelector